src/Providers/LoadConfigServiceProvider.php

```diff
@@ -26,6 +26,11 @@ public function boot()
             // Don't throw any errors, we may not be setup yet.
         }
 
+        // Set the Segment.com settings.
+        $repository = $this->app->make('CachetHQ\Cachet\Segment\RepositoryInterface');
+        $segmentWriteKey = $repository->fetch();
+        $this->app->config->set('cachethq/segment::write_key', $segmentWriteKey);
```

Comment:
`$segmentRepository = $this->app->make('CachetHQ\Cachet\Segment\RepositoryInterface');`
`$this->app->config->set('cachethq/segment::write_key', $segmentRepository->fetch());`